src/main/resources/org/sagebionetworks/web/client/widget/sharing/AccessControlListModalWidgetViewImpl.ui.xml

```diff
@@ -18,7 +18,7 @@
 			<w:HelpWidget
 				helpMarkdown="Sharing settings determine who can access your content, and what kind of access they have. Choose people/teams and define their level of access below. &#10;&#10;_Only Administrators can add, delete, or change access levels for other people._"
 				addStyleNames="margin-left-5" placement="BOTTOM"
-				href="http://docs.synapse.org/articles/access_controls.html" />
+				href="https://help.synapse.org/docs/Sharing-Settings-and-Permissions.2024276030.html" />
 		</b:ModalHeader>

```

Comment:
Double-check: I have https://help.synapse.org/docs/Conditions-for-Use.2009596938.html in spreadsheet..

Reply:
The target 2024276030 is what I have for /articles/sharing_settings. Similar, need to look at it in-place.

Reply:
Yes, this was a bug.  In the ACL editor, if you went to the help page on the Sharing Settings dialog (ACL modal), it incorrectly takes you to information about conditions for use.  It should send you to information about the Sharing Settings.